src/app/common/use-wallet-type.ts

```diff
@@ -14,7 +14,7 @@ function isSoftwareWallet(walletType: WalletType) {
 
 type WalletTypeMap<T> = Record<WalletType, T>;
 
-function whenWallet(walletType: WalletType) {
+function whenWallet(walletType: WalletType = 'ledger') {
```

Comment:
Maybe this works for now, but we do want to ultimately remove

Reply:
yeah. not sure why added this actually🤔